src/Compiler/CompilerPasses.cpp

```diff
@@ -121,17 +122,33 @@ void addONNXToKrnlPasses(mlir::PassManager &pm, int optLevel, bool enableCSE,
   if (enableInstrumentONNXSignature)
     pm.addNestedPass<func::FuncOp>(
         onnx_mlir::createInstrumentONNXSignaturePass());
+  if (enableLinalg) {
+    pm.addPass(onnx_mlir::createLowerONNXToLinalgPass());
+    // Linalg bufferization can be before or after LowerToKrnlPass
+    // However, sice bufferization::AllocTensorOp is in LowerToKrnl temporarily,
+    // these passes have to be called before LowerToKrnlPass
+    pm.addNestedPass<func::FuncOp>(createLinalgBufferizePass());
```

Comment:
Using `OneShotBufferization` is the recommended way to do bufferization now rather than calling individual bufferize passes.

Reply:
I just tried the OneShotBufferization and got the following error:
```
loc("run-matmul/matmul-dyn.mlir":4:8): error: op was not bufferized
// -----// IR Dump After OneShotBufferize Failed (one-shot-bufferize) //----- //
```
Could you tell me how to use the OneShotBufferization? I add the pass before lowering to Krnl with this line of code:
```
pm.addPass(mlir::bufferization::createOneShotBufferizePass());
```